### PR TITLE
Add support for GeoJSON

### PR DESCRIPTION
**Pull request by sharlagelfand**
Adding support for GeoJSON! I think I covered everything (added an example, added geojsonio to suggests, reran codemeta, ran docs, etc). But let me know if I missed anything.

Also, thank you for adding me as an author! ☺️

### CODE REVIEW

File: R/ckan_fetch.R [hunk 1]
```diff
@@ -50,6 +50,12 @@
 #' x <- ckan_fetch(res$url)
 #' class(x)
 #' plot(x)
+#'
+#' GeoJSON file - requires geojsonio package
```
Comment (sckott) [inline R/ckan_fetch.R]: make sure to comment out comments in the examples

Comment (sharlagelfand) [reply R/ckan_fetch.R]: oops! thank you!

**Comment by sharlagelfand** [comment]
honestly i don't know much about GeoJSON and just googled packages for it -- used `geojsonio` for the obvious reason without realizing that `sf` could handle GeoJSON too! i agree, since `sf` is already used it makes sense to use that too. i'll update the code.

ps, i'm working on [this package](https://github.com/sharlagelfand/opendatatoronto) hence my sudden `ckanr` investment 😂 thank you for being so receptive to these changes!

**Comment by sharlagelfand** [comment]
sorry for the messy commits 🙈 rebasing is hard

**Comment by sckott** [comment]
thanks, looks good. cool about the package (see also https://github.com/ropensci/ckanr/issues/113 and https://github.com/VLucet/rgovcan - also canada) 

just one change, can you undo the codemeta.json file cahange, so we can keep the PR focused on the actual changes here

**Comment by sharlagelfand** [comment]
done, thank you! thanks for linking that package too!

**Comment by sckott** [comment]
thanks @sharlagelfand

**Pull request merged**